clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java

```diff
@@ -233,14 +228,7 @@ public void stop() {
 
     entity = null;
 
-    try {
-      if (clusterConnection != null) {
-        clusterConnection.close();
-        clusterConnection = null;
-      }
-    } catch (IOException ex) {
-      throw new RuntimeException(ex);
-    }
+    closeConnection();
```

Comment:
Note: change of behavior: before, the IOException was rethrown. I don't know the reason because all other places just logs it, and if it is rethrown, it will be rethrown at CM.close() and this prevents the transition listeners to be called, whereas all the services are closed.